modules/tinymce/src/core/test/ts/browser/EditorTest.ts

```diff
@@ -45,6 +45,24 @@ describe('browser.tinymce.core.EditorTest', () => {
     editor.off('change');
   });
 
+  it('TINY-6920: Do not fire change event at first typed character', () => {
```

Comment:
Should this be in the UndoManagerTest unsure.

Reply:
I'd agree this is more suited to an undo manager test, so it should be somewhere closer to there.